setup.py

```diff
@@ -1,8 +1,78 @@
 import setuptools
 import os
+import subprocess
 from os import path
 import sys
 
+current_dir = os.path.dirname(os.path.realpath(__file__))
+
+build_dir = os.path.join(current_dir, 'deps_build')
+
+if not os.path.exists(build_dir):
+    os.mkdir(build_dir)
+
+os.chdir(build_dir)
+dep_install_path = os.getenv('AWS_C_INSTALL')
```

Comment:
new users aren't going to have this environment variable. Should we just make a local install directory, like we make a local build directory?